### PR TITLE
[BPK-1107] Add docs for native switch component

### PR DESCRIPTION
**Pull request by shaundon**
0 0 0 0-8080-components-native-switch

### CODE REVIEW

File: packages/bpk-docs/package-lock.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
-	"requires": true,
+	"name": "bpk-docs",
```
Comment (matthewdavidson) [inline packages/bpk-docs/package-lock.json]: This `package-lock.json` has the package name in it again  😤 

I still do not know what causes this.

File: packages/bpk-docs/src/constants/routes.js [hunk 2]
```diff
@@ -76,10 +76,11 @@ export const NATIVE_BANNER_ALERT = '/components/native/banner-alerts';
 export const NATIVE_BUTTON = '/components/native/button';
 export const NATIVE_CARDS = '/components/native/cards';
 export const NATIVE_COMPONENTS = '/components/native';
-export const NATIVE_ICONS = '/components/native/icons';
 export const NATIVE_HORIZONTAL_NAV = '/components/native/horizontal-nav';
-export const NATIVE_SPINNER = '/components/native/spinners';
+export const NATIVE_ICONS = '/components/native/icons';
 export const NATIVE_INPUT = '/components/native/text-input';
+export const NATIVE_SPINNER = '/components/native/spinners';
+export const NATIVE_SWITCH = '/components/native/switch';
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/constants/routes.js]: This should be plural.

File: packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js [hunk 3]
```diff
@@ -75,15 +75,15 @@ const links = [
       { id: 'NATIVE_BANNER_ALERT', route: routes.NATIVE_BANNER_ALERT, children: 'Banner alerts' },
       { id: 'NATIVE_BUTTON', route: routes.NATIVE_BUTTON, children: 'Button' },
       { id: 'NATIVE_CARDS', route: routes.NATIVE_CARDS, children: 'Cards' },
+      { id: 'NATIVE_HORIZONTAL_NAV', route: routes.NATIVE_HORIZONTAL_NAV, children: 'Horizontal navigation' },
       { id: 'NATIVE_ICONS', route: routes.NATIVE_ICONS, children: 'Icons' },
       { id: 'NATIVE_INPUT', route: routes.NATIVE_INPUT, children: 'Text input' },
       { id: 'NATIVE_SPINNER', route: routes.NATIVE_SPINNER, children: 'Spinners' },
-      { id: 'NATIVE_HORIZONTAL_NAV', route: routes.NATIVE_HORIZONTAL_NAV, children: 'Horizontal navigation' },
+      { id: 'NATIVE_SWITCH', route: routes.NATIVE_SWITCH, children: 'Switch' },
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js]: Again, this should be plural. Btw there is sort function on the docs nav so you don't need to manually sort these ... 😛

File: packages/bpk-docs/package-lock.json (hunk 1 above)
Comment (shaundon) [reply packages/bpk-docs/package-lock.json]: 😭 not again!

File: packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js (hunk 3 above)
Comment (shaundon) [reply packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js]: Haha, I know it sorts them manually but I couldn't leave it unsorted in the code! I realise that's ridiculous though haha

File: packages/bpk-docs/package-lock.json (hunk 1 above)
Comment (matthewdavidson) [reply packages/bpk-docs/package-lock.json]: Notice that my PR remvoes them -> https://github.com/Skyscanner/backpack/pull/256/files (cmd + f "package-lock.json", there are two examples)

Comment (shaundon) [reply packages/bpk-docs/package-lock.json]: This page mentions, or at least implies, that the name will be part of the lockfile: https://docs.npmjs.com/files/package-locks (it's about halfway down the page).

Still no idea why it's happening just to me but at least we know it's a thing..

**Pull request merged**